src/common/textures/gametexture.cpp

```diff
@@ -362,7 +372,7 @@ void FGameTexture::SetSpriteRect()
 			spi.mSpriteRect.width = (float)spi.spriteWidth / fxScale;
 			spi.mSpriteRect.height = (float)spi.spriteHeight / fyScale;
 
-			if (spi.mTrimResult > 0)
+			if (spi.mTrimResult > 0 && r_trimspriteborders)
```

Comment:
This changes the default behavior because `r_trimspriteborders` has the initial value set to `false`. Is it intentional?

Reply:
Yes. This is based on what I [posted here.](https://forum.zdoom.org/viewtopic.php?f=18&t=73072#p1207596) (see Graf & Rachael's comments)